qdrant_client/local/local_collection.py

```diff
@@ -879,6 +878,13 @@ def _query_collection(
                 with_vectors=with_vectors,
             )
             return [record_to_scored_point(record) for record in records[offset:]]
+        elif query == models.SampleQuery(sample=models.Sample.RANDOM):
```

Comment:
just nit: seems to be a little bit odd, maybe it's better to isinstance + ==models.Sample.RANDOM?